scripts/run-integration-tests.sh

```diff
@@ -31,7 +31,7 @@ ARCH=$(go env GOARCH)
 : "${RUNNING_PERFORMANCE:=false}"
 : "${RUN_CALICO_TEST:=false}"
 : "${RUN_LATEST_CALICO_VERSION:=false}"
-: "${CALICO_VERSION:=3.22.0}"
+: "${CALICO_VERSION:=v3.23.0}"
```

Comment:
here there is v3.23.0 but in scripts/run-cni-release-tests.sh it is without a "v" so is it a typo maybe?

Reply:
I have tested calico from running ```./scripts/run-integration-tests.sh```, but not tested ```run-cni-release-tests.sh```. Thus I didn't append the "v" as a prefix. Since they seem using the same test suite to run Calico test, I think it is ok to do the same for consistence. Let me add it as well. Thanks for pointing it out.